### PR TITLE
Added base logout view

### PR DESCRIPTION
**Pull request by clintonb**
ECOM-4607

### CODE REVIEW

**Comment by clintonb** [comment]
FYI @edx/ecommerce

File: auth_backends/views.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    This is a base view. You MUST override `url` attribute or `get_redirect_url` method to make this view useful. See
+    the documentation for `RedirectView` for more info:
+    https://docs.djangoproject.com/en/1.9/ref/class-based-views/base/#redirectview.
+
+    If the `no_redirect` querystring argument is set, the view will not redirect. An empty 200 response will be
+    returned instead.
+
+    This view is intended to be used with Django projects that need to implement single sign out/logout. After
+    destroying their session is destroyed, the user will be redirected to the specified logout page on the
```
Comment (rlucioni) [inline auth_backends/views.py]: ~~destroying~~

Comment (clintonb) [inline auth_backends/views.py]: Updated.

**Comment by rlucioni** [comment]
👍

**Pull request merged**